include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -43,10 +43,18 @@
 #if USE_PARTITION_MANAGER
 #include <pm_config.h>
 #ifdef PM_ADDRESS_LOAD_OFFSET
+/* We are linking mcuboot against S1, S0 is Image 1 primary */
+_image_1_primary_slot_id = PM_S0_ID;
```

Comment:
Where is the code which uses this symbol located?

Reply:
It will be in mcuboot

Reply:
Okei.